xdsl/transforms/arith_add_fastmath.py

```diff
@@ -32,18 +32,20 @@ class AddArithFastMathFlags(RewritePattern):
     @op_type_rewrite_pattern
     def match_and_rewrite(
         self,
-        op: arith.Addf
-        | arith.Subf
-        | arith.Mulf
-        | arith.Divf
-        | arith.Minimumf
-        | arith.Maximumf,
+        op: (
+            arith.Addf
+            | arith.Subf
+            | arith.Mulf
+            | arith.Divf
+            | arith.Minimumf
+            | arith.Maximumf
+        ),
```

Comment:
I think that is an unrelated change? At least, that's not usually the style we are using

Reply:
There's some weirdness in Black formatting, I didn't make this change manually. I've seen it in some other PRs, I think I have a situation where the formatter in VSCode and in precommit are slightly different, and both slightly ambiguous

Reply:
#2271